docs/tutorial.md

```diff
@@ -2,14 +2,18 @@
 
 ## Prerequisites
 
-You must have a Kubernetes cluster. Additionally, you will need a local configured copy of `kubectl`. This example requires more resources then the [quick start](quickstart.md) tutorial, therefore you will need something larger then a typical minikube cluster. A four node cluster with 36 total vCPUs (8 on each node) and 64GB total memory (16GB on each node) is generally sufficient.
+* Kubernetes Cluster
+* `kubectl` properly configured
+* [kustomize](https://github.com/kubernetes-sigs/kustomize/releases) (v3.1.0+)
+* [redskyctl](https://github.com/redskyops/redskyops-controller/releases)
+* Red Sky Ops controller running ( `redskyctl init` )
 
-A local install of [Kustomize](https://github.com/kubernetes-sigs/kustomize/releases) (v3.1.0+) is required to manage the objects in you cluster.
-
-Additionally, you will to initialize Red Sky Ops in your cluster. You can download a binary for your platform from the [releases page](https://github.com/redskyops/redskyops-controller/releases) and run `redskyctl init` (while connected to your cluster). For more details, see [the installation guide](install.md).
+This example requires more resources then the [quick start](quickstart.md) tutorial, therefore you will need something larger then a typical minikube cluster. A four node cluster with 32 total vCPUs (8 on each node) and 64GB total memory (16GB on each node) is generally sufficient.
 
 ## Tutorial Resources
 
+######### TODO ########### tutorial does not exist, should this be elasticsearch?
```

Comment:
Want to make sure this gets caught prior to release

Reply:
Yep, let's use elasticsearch as the tutorial